### PR TITLE
[[ LCB ]] Add 'my ... paint' syntax

### PR DESCRIPTION
**Pull request by runrevmark**
This PR adds 'my ... paint' syntax to LCB. This allows widgets to fetch a paint to use based on the standard color and pattern properties set in the host environment.

### CODE REVIEW

**Comment by runrevmark** [comment]
@livecodeali: That was an error - fixed now.

**Comment by livecodeali** [comment]
@livecode-vulcan review ok cc92c0c

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok cc92c0ca14fe39b643ebaea13a8c2d0caf9dac3d

**Comment by livecode-vulcan** [comment]
😎 test success cc92c0ca14fe39b643ebaea13a8c2d0caf9dac3d
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/1986)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/1999)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/1980)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/1992)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/2002)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/2008)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/2012)

File: engine/src/widget-syntax.cpp [hunk 1]
```diff
@@ -262,6 +262,56 @@ extern "C" MC_DLLEXPORT_DEF void MCWidgetGetMyDisabled(bool& r_disabled)
...
+    
+    // If getforecolor() returns true then we have a solid color.
+    MCColor t_color;
+    MCPatternRef t_pattern = nil;
+    int2 x, y;
+    if (t_host -> getforecolor((uint2)p_type, False, False, t_color, t_pattern, x, y, CONTEXT_TYPE_SCREEN, t_host, False))
+    {
+        MCAutoValueRefBase<MCCanvasColorRef> t_canvas_color;
+        if (!MCCanvasColorCreateWithRGBA(t_color . red / 65535.0f, t_color . blue / 65535.0f, t_color . green / 65535.0f, 1.0f, &t_canvas_color))
+            return;
```
Comment (livecodefraser) [inline engine/src/widget-syntax.cpp:281]: RGB, not RBG, please! :stuck_out_tongue:

**Pull request merged**